### PR TITLE
Create alteonos.yaml

### PR DESCRIPTION
**Pull request by sfini**
Add Support for radware/Alteon application Switch

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by murrant** [comment]
You'll need to add a snmprec file for each entry in your discovery array.

**Comment by laf** [comment]
@sfini Please also remove the comments from the yaml file.

**Comment by sfini** [comment]
@murrant 
Is snmpwalk ok ?

**Comment by laf** [comment]
@sfini We don't need snmpwalk, you just need to create a test unit file as per: http://docs.librenms.org/Developing/os/Test-Units/

**Comment by sfini** [comment]
@murrant 
Hi Tony .. Added radware.snmprec ...  Is it ok ?

**Comment by sfini** [comment]
@murrant 

https://github.com/librenms/librenms/blob/aff2bb084f0008664bbadd2a14e6bd27a7de9a97/tests/snmpsim/radware.snmprec

**Comment by laf** [comment]
You've not pushed the file into this branch but into a new one :)

In https://github.com/sfini/librenms - Click on the master branch in the top left:

image

Select patch-1 branch, now add the file to that and it should push it here.

**Comment by sfini** [comment]
@murrant 

All Done ... Thank you..

File: includes/definitions/radware.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+os: radware
+text: RADWARE
+type: network
+icon: radware
```
Comment (laf) [inline includes/definitions/radware.yaml]: Sorry, one last thing I've noticed is that the icon radware doesn't exist. You will also need to add this in:

http://docs.librenms.org/Developing/os/Initial-Detection/#icon-and-logo

**Comment by sfini** [comment]
@laf 
Done...

File: includes/definitions/radware.yaml (hunk 1 above)
Comment (sfini) [inline includes/definitions/radware.yaml]: Done :+1:

**Comment by laf** [comment]
One last question, is radware the right name for the os? That seems to be the vendor name which we don't usually use. I.e Cisco have Cisco IOS so the os name is ios. What's the case here?

**Comment by sfini** [comment]
@laf 
Hi Neil .. You're right .. The correct OS for this type is AlteonOS ...
There are another branches of RADWARE co. , that have other OS (like APSoluteOS, etc.)
So, i think is better to have different type of yaml files as you have for cisco ....
I have renamed radware.yaml in alteonos.yaml, and made all the changes.
I also pushed the alteonos.rb to the oxidized's github.

**Comment by sfini** [comment]
@laf 
Do i have to upload mib too ?

**Comment by laf** [comment]
If you aren't making use of the mib (you aren't so far) then no.

**Pull request merged**